api/profiles.yaml

```diff
@@ -76,25 +83,110 @@ commands:
         type: integer
         property: oeeTargets.availability
         description: OEE target availability
-        default: 80
 
       - name: performance
         type: integer
         property: oeeTargets.performance
         description: OEE target performance
-        default: 80
 
       - name: overall
         type: integer
         property: oeeTargets.overall
         description: OEE target overall
-        default: 80
       
       - name: quality
         type: integer
         property: oeeTargets.quality
         description: OEE target quality
-        default: 80
+
+    bodyTemplates:
+      - type: jsonnet
```

Comment:
how does this template definition relate to the "profile.jsonnet" file? is that just an example or is that really used at runtime?

Reply:
no, this is a template that is always applied to the body of the HTTP request and if users do not define any changes, the request body will be the same as in this bodyTemplates section.
This template doesn't have a relation to the template file